.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+## Summary
+<!--
+    Provide a link to the ticket or document which prompted this change,
+    Describe the rationale and design decisions.
+-->
+
+## Feature Flag
+<!-- If this is specific to a feature flag, which one? -->
+
+## Product Description
+<!-- For non-invisible changes, describe user-facing effects. -->
+
+## Safety Assurance
+
+- [ ] If the PR is high rish, "High Risk" label is set
+- [ ] I have confidence that this PR will not introduce a regression for the reasons below
+- [ ] Do we need to enhance manual QA test coverage ? If yes, "QA Note" label is set correctly
+
+### Automated test coverage
+
+<!-- Identify the related test coverage and the tests it would catch -->
+
+
+### Safety story
+<!--
+Describe any other pieces to the safety story including
+local testing, why the change is inherently safe, and/or plans to limit the blast radius of a defect.
+-->
+
+### Rollback instructions
```

Comment:
I'm not sure if we rollback PRs. I think we mostly fix the issues in the hotfixes, so this can probably be removed, right?

Reply:
makes sense